### ISSUE TITLE
[T3][Accessibility][UI] Drop down doesnot work

### ISSUE THREAD

**Issue by naveennayakp** [Mohanme13/Feb-19-Bug-bash#63]
In Social history tab for the field next to frequency, the drop down doesnot work with arrow buttons

**Comment by naveennayakp** [comment]
[bug1.docx](https://github.com/Mohanme13/Feb-19-Bug-bash/files/137905/bug1.docx)
